huggingface_sb3/naming_schemes.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
Maybe a better name for the file is `naming_utils.py`. WDYT?

Reply:
No strong opinion on this. I am fine with both. I slightly think, that the term `utils` is a bit overused and tried to use something more specific.